css-pseudo-classes/css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,19 @@
+button {
+  width: 170px;
+  height: 50px;
+  text-size-adjust: 10;
+}
+
+button:hover {
+  background-color: crimson;
+  cursor: pointer;
+}
```

Comment:
Please make the button colors the same as in the animation:

---

![image](https://user-images.githubusercontent.com/16143771/151229343-cdaa037c-f18e-4642-88be-72d228a89a96.png)